test_scripts/Resumption/InteriorVehicleData/001_InteriorVD_resumption_one_moduleType_without_moduleId_unexpected_disconnect.lua

```diff
@@ -0,0 +1,59 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data after transport disconnect
+--  in case app was subscribed to one module without moduleId
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1 via GetInteriorVehicleData(module_1)
```

Comment:
moduleType_1 ? or module_1?

Reply:
@yang1070 It is has been fixed in all the scripts of this PR.